clients/admin-ui/src/features/privacy-requests/RequestTable.tsx

```diff
@@ -113,13 +107,12 @@ export const RequestTable = ({ ...props }: BoxProps): JSX.Element => {
   const handleSort = (columnSort: ColumnSort) => {
     if (!columnSort) {
       dispatch(clearSortKeys());
-      resetPageIndexToDefault();
```

Comment:
@lucanovera Do you have any insight as to when / where this reset scenario would matter? This reset was what was causing the page and size not to stay in the URL / page.

Reply:
The pagination resets without this line so I'm going to assume it's not longer necessary.